src/MetisGraphPartitioner.cpp

```diff
@@ -149,7 +161,7 @@ void MetisGraphPartitioner::SetPostIsolateVertices(const mfem::Array<int>& indic
 }
 
 mfem::SparseMatrix MetisGraphPartitioner::getAdjacency(
-    const mfem::SparseMatrix& wtable) const
+    const mfem::SparseMatrix& wtable)
```

Comment:
Why remove the `const`?

Reply:
The function is now static

Reply:
Yea, maybe this function should be taken out as an independent function (something like `RemoveDiagonal` in `MatrixUtilities.xpp`)?